its/src/main/resources/META-INF/plugin.xml

```diff
@@ -26,7 +26,7 @@
   <vendor url="https://sonarsource.com">SonarSource</vendor>
   <description>Fake Plugin To Run SonarLint UI tests with multiple flavors</description>
 
-  <idea-version since-build="213.5744.223"/>
+  <idea-version since-build="223"/>
```

Comment:
I think we should set a more precise version here, else we don't know exactly against which version we build